Scripts/RemediationScripts/ControlsEligibleForRemediationThroughUI.md

```diff
@@ -561,4 +563,52 @@ Immutability must be enabled and locked on Recovery Services Vault
 
 ### Minimum permissions required to run the script
 Contributor or higher privileged role on the Redis Enterprise
+___
+
+## 35. Azure_BackupVault_DP_Enable_Immutability
+
+### Display Name
```

Comment:
There should be single entry in this file for this control, recheck please

Reply:
Removed duplication